test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -4202,6 +4208,361 @@ defmodule Trento.SapSystemTest do
     end
   end
 
+  describe "delta deregistration" do
```

Comment:
I wouldn't call this `delta deregistration`.
Maybe `instance marked absent/present` or similar?